src/Polytopes/Cone.jl

```diff
@@ -35,57 +88,43 @@ end
 Base.eltype(::Type{ConeRayIterator}) = Polymake.VectorAllocated{Polymake.Rational}
 Base.length(iter::ConeRayIterator) = n_rays(iter.cone)
 
-"""
-   n_rays(C::Cone)
-
-Returns the number of rays of the cone `C`.
-"""
-n_rays(C::Cone) = size(pm_cone(C).RAYS, 1)
-
 """
    rays(C::Cone)
 
 Returns the rays of a cone.
 """
 rays(C::Cone) = ConeRayIterator(C)
 
+###############################################################################
+###############################################################################
+### Access properties
+###############################################################################
+###############################################################################
 
-"""
-    positive_hull(generators::Union{Oscar.MatElem,AbstractMatrix})
+###############################################################################
+## Scalar properties
+###############################################################################
 
-A polyhedral cone, not necessarily pointed, defined by the positive hull
-of the `generators`. Redundant rays are allowed in the generators.
 """
-function positive_hull(generators::Union{Oscar.MatElem,AbstractMatrix})
-    # TODO: Filter out zero rows
-    C=Polymake.polytope.Cone{Polymake.Rational}(INPUT_RAYS =
-      matrix_for_polymake(remove_zero_rows(generators)))
-    Cone(C)
-end
-
-#Is there a Polymake.jl function for this?
-function pm_set_to_julia(S::Polymake.SetAllocated{Int64})
-    return(Array{Int64,1}(S))
-end
-
+   n_rays(C::Cone)
 
+Returns the number of rays of the cone `C`.
+"""
+n_rays(C::Cone) = size(pm_cone(C).RAYS, 1)
```

Comment:
Nitpick: Julia has `ndims`, `ndigits`, `ncodeunits` and in Oscar we have `ngens`, `ncols`, `nrows`, ... -- so by that pattern, it ought to be `nrays`, `nfacets`, `nvertices`

Reply:
`nmaximal_cones` or `nmaximalcones`?

Reply:
`nmaximal_cones` I would say.  Although ugly, let's try to be consistent.